Editor/Module Management/ModuleInstaller.cs

```diff
@@ -101,12 +100,6 @@ public static void AddModuleRequest(string identifier)
         {
             var startTime = Time.realtimeSinceStartup;
             AddRequest addRequest = Client.Add(identifier);
-
-            // reset first time setup flag to ensure setup guide window is displayed
-            if (identifier.Contains(AVATAR_LOADER_SUBSTRING))
-            {
-                ProjectPrefs.SetBool(ProjectPrefs.FIRST_TIME_SETUP_DONE, false);
-            }
```

Comment:
When was this reset or why was it necessary?

Reply:
Tbh I don't remember why exactly, but it was used for the separate AvatarLoader package, and since that's no longer required (due to 3.0 core release) I removed.